packages/map/src/Utility.ts

```diff
@@ -343,11 +413,13 @@ export function albersUsaPr() {
     // The projections must have mutually exclusive clip regions on the sphere,
     // as this will avoid emitting interleaving lines and polygons.
     albersUsa.stream = function (stream) {
+        return cache && cacheStream === stream ? cache : cache = multiplex([lower48.stream(cacheStream = stream), alaska.stream(stream), hawaii.stream(stream)]);
```

Comment:
No Puerto Rico?

Reply:
Didn't it get blown away last summer?
(good catch - thx!)